js/src/forum/components/DiscussionPage.js

```diff
@@ -37,7 +38,7 @@ export default class DiscussionPage extends Page {
     // page, then we don't want Mithril to redraw the whole page – if it did,
     // then the pane would which would be slow and would cause problems with
     // event handlers.
-    if (app.cache.discussionList) {
+    if (app.cache.discussionList.hasDiscussions()) {
```

Comment:
This seems like slightly different behavior - "the discussion list being loaded" (i.e. present) is a different condition from "the discussion list having discussions". An empty list can be loaded as well, right?

Reply:
I think this was brought up by David in another place before, so I might be misguided pointing this out again here.

Reply:
Correct, I brought it up @ https://github.com/flarum/core/pull/2150#discussion_r422225696.

Reply:
Ah, right. This is the reverse condition, though, and here we now have an unhandled case of "results loaded, but none found"...

Reply:
I don't see any benefit to showing the slideout when there's no discussions?